src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    public static final SessionFactory instance = initSessionFactory();
```

Comment:
It's a good practice to use a more thread-safe approach to create a singleton pattern with lazy initialization, such as the 'Initialization-on-demand holder idiom'.